proto/regen/data/v1alpha1/query.proto

```diff
@@ -9,20 +9,20 @@ option go_package = "github.com/regen-network/regen-ledger/x/data";
 // Query is the regen.data.v1alpha1 Query service
 service Query {
 
-    // Data queries data based on its CID.
-    rpc Data(QueryDataRequest) returns (QueryDataResponse);
+    // ByCid queries data based on its CID.
+    rpc ByCid(QueryByCidRequest) returns (QueryByCidResponse);
```

Comment:
This change is not necessarily needed, but I think it's clearer, esp given that the module name is already data